resonate/models/context.py

```diff
@@ -31,15 +39,21 @@ class LFX:
     opts: Options = field(default_factory=Options)
     versions: dict[int, Callable] | None = None
 
-    def options(self, *, id: str | None = None, send_to: str | None = None, timeout: int | None = None, version: int | None = None, tags: dict[str, str] | None = None) -> Self:
+    def __post_init__(self) -> None:
+        self._initial_timeout = self.opts.timeout
+
+    def options(self, *, id: str | None = None, send_to: str | None = None, timeout: int | None = None, version: int | None = None, tags: dict[str, str] | None = None, retry_policy: RetryPolicy | None = None) -> Self:
```

Comment:
Super anal, but could we declare the params alphabetically? Also down to split them over multiple lines